### DISCUSSION TITLE
Virtual scrolling grouped infinite scrolling is not working

### DISCUSSION THREAD

**Discussion by manikandanYsecIT** [komarovalexander/ka-table#452] [General]
Virtual scrolling grouped infinite scrolling is not working properly. Only the initial page size comes after that, not scrolling. Also if I enable virtualScrolling={{enabled: true}} table is working very slowly.

**Comment by komarovalexander** [comment]
Hi @AlexYsec , can you reproduce it on https://stackblitz.com/edit/table-many-rows-grouping-ts?file=Demo.tsx thanks

**Reply by manikandanYsecIT** [reply]
If I give virtualScrolling={{enabled: true}} for table then page performance too low. Also not working on Details Row table.
